### PR TITLE
Added rto

### PR DESCRIPTION
**Pull request by ankitdas13**
Doc ref (Updated)

[View RTO Reasons](https://razorpay.com/docs/payments/magic-checkout/rto-intelligence/#step-2-view-rto-risk-reasons)
[Update fulfilment details](https://razorpay.com/docs/payments/magic-checkout/rto-intelligence/#step-3-update-the-fulfillment-details)

### CODE REVIEW

**Comment by shwatang** [comment]
> [View RTO Reasons](https://betasite.razorpay.com/docs/razorpay/IN/payments-magic-new-rto-intelligence/payments/magic-checkout/rto-intelligence/#12-view-rto-risk-reasons)
[Update fulfilment details](https://betasite.razorpay.com/docs/razorpay/IN/payments-magic-new-rto-intelligence/payments/magic-checkout/rto-intelligence/#13-update-the-fulfillment-details)

Please update the links. Betasite links provided.

File: src/Entity.cs [hunk 1]
```diff
@@ -15,6 +15,7 @@ public class Entity
             {"payment.downtime", "Razorpay.Api.Payment" },
             {"refund", "Razorpay.Api.Refund"},
             {"order", "Razorpay.Api.Order"},
+            {"order.fulfillment", "Razorpay.Api.Order"},
```
Comment (shwatang) [inline src/Entity.cs:18]: Didn't get this. How is this needed?

File: src/Order.cs [hunk 2]
```diff
@@ -70,5 +70,34 @@ public Order Edit(Dictionary<string, object> data)
                 return null;
             }
         }
+
+        public Order ViewRtoReview(Dictionary<string, object> data=null)
+        {
+            string relativeUrl = string.Format("{0}/{1}/{2}/rto_review", GetUrlVersion(), GetEntityUrl(), this["id"]);
+            List<Entity> entities = Request(relativeUrl, HttpMethod.POST, data);
```
Comment (shwatang) [inline src/Order.cs:77]: Is this a POST request?

**Review by shwatang — COMMENTED** [review]
Please address the comments.

File: src/Entity.cs (hunk 1 above)
Comment (ankitdas13) [reply src/Entity.cs:18]: according to this response 
https://razorpay.com/docs/payments/magic-checkout/rto-intelligence/#step-3-update-the-fulfillment-details
```
{
  "entity": "order.fulfillment",
  "order_id": "EKwxwAgItXXXX",
  "payment_method": "upi",
  "shipping": {
    "waybill": "123456789",
    "status": "rto",
    "provider": "Bluedart"
  }
}
```

**Comment by ankitdas13** [comment]
> > [View RTO Reasons](https://betasite.razorpay.com/docs/razorpay/IN/payments-magic-new-rto-intelligence/payments/magic-checkout/rto-intelligence/#12-view-rto-risk-reasons)
> > [Update fulfilment details](https://betasite.razorpay.com/docs/razorpay/IN/payments-magic-new-rto-intelligence/payments/magic-checkout/rto-intelligence/#13-update-the-fulfillment-details)
> 
> Please update the links. Betasite links provided.

Update the doc

File: src/Order.cs (hunk 2 above)
Comment (ankitdas13) [reply src/Order.cs:77]: Yes it is a post request

Screenshot 2024-05-06 at 1 26 12 PM

**Pull request merged**